doc/classes/CPUParticles2D.xml

```diff
@@ -68,132 +92,188 @@
+			Orbital velocity randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="preprocess" type="float" setter="set_pre_process_time" getter="get_pre_process_time">
+			Particle system starts as if it had already run for this many seconds.
 		</member>
 		<member name="radial_accel" type="float" setter="set_param" getter="get_param">
+			Radial acceleration applied to each particle. Makes particle accelerate away from origin.
 		</member>
 		<member name="radial_accel_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's radial acceleration will vary along this [Curve].
 		</member>
 		<member name="radial_accel_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Radial acceleration randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="randomness" type="float" setter="set_randomness_ratio" getter="get_randomness_ratio">
+			Emission lifetime randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="scale_amount" type="float" setter="set_param" getter="get_param">
+			Initial scale applied to each particle.
 		</member>
 		<member name="scale_amount_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's scale will vary along this [CurveTexture].
 		</member>
 		<member name="scale_amount_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Scale randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="speed_scale" type="float" setter="set_speed_scale" getter="get_speed_scale">
+			Particle system's running speed scaling ratio. Default value: [code]1[/code]. A value of [code]0[/code] can be used to pause the particles.
 		</member>
 		<member name="spread" type="float" setter="set_spread" getter="get_spread">
+			Each particle's initial direction range from [code]+spread[/code] to [code]-spread[/code] degrees. Default value: [code]45[/code].
 		</member>
 		<member name="tangential_accel" type="float" setter="set_param" getter="get_param">
+			Tangential acceleration applied to each particle. Tangential acceleration is perpendicular to the particle's velocity giving the particles a swirling motion.
 		</member>
 		<member name="tangential_accel_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's tangential acceleration will vary along this [Curve].
 		</member>
 		<member name="tangential_accel_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Tangential acceleration randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="texture" type="Texture" setter="set_texture" getter="get_texture">
+			Particle texture. If [code]null[/code] particles will be squares.
 		</member>
 	</members>
 	<constants>
-		<constant name="DRAW_ORDER_INDEX" value="0" enum="DrawOrder">
```

Comment:
These constants should stay, they're used in the source code and for the `draw_order` property.

Reply:
Oops!